vendor/vendor.json

```diff
@@ -105,5 +117,5 @@
 			"revisionTime": "2017-07-27T04:22:29Z"
 		}
 	],
-	"rootPath": "github.com/prometheus/graphite_exporter"
+	"rootPath": "github.com/travisgroth/graphite_exporter"
```

Comment:
We'll need to change this back before it hits master … I think one way to be able to develop locally under a different name is to check out the exporter at github.com/prometheus/graphite_exporter and add your fork as an additional remote.

Reply:
I probably broke the vendor.json rootPath early on so let me fix that and repath.

I...think...that will also fix the promu build error locally but I don't think the forked branch will build in CI until its in the home repo.  I have no experience with promu but it seems to be picking up the repo path from a yaml file.  Any advice there?